syntax = "proto3";

package voice;

service VoiceRecognizer {
    rpc GetText(GetTextRequest) returns (GetTextResponse) {};
}

message GetTextRequest {
    bytes stream = 1;
    float bitrate = 2;
    int32 sampleRate = 3;
}

message GetTextResponse {
    string message = 1;
}